bundle/src/main/java/com/adobe/acs/commons/mcp/impl/processes/asset/FileOrRendition.java

```diff
@@ -203,8 +205,7 @@ public long getLength() throws IOException {
                 try {
                     size = getConnection().getContentLengthLong();
                 } catch (IOException ex) {
-                    Logger.getLogger(FileOrRendition.class.getName()).log(Level.SEVERE, null, ex);
```

Comment:
We should log this properly too, I think.  Errors in this class don't always bubble up where they can be inspected easily.